rpmconf/Upgrade/zmupgrade.pm

```diff
@@ -407,6 +407,7 @@ sub upgrade {
       if ($schema_found) {
         $needMysqlUpgrade=1 if ($v eq "8.0.0_GA");
         $needMysqlUpgrade=1 if ($v eq "8.5.0_BETA1");
+        $needMysqlUpgrade=1 if ($v eq "8.7.10_GA");
```

Comment:
Hmm, yeah, the mariadb 10.0 to 10.1 happened with 8.7.0 ($v eq "8.7.0_BETA1").
Doing it for 8.7.10_GA instead is fine, but did that window close already?  This change won't make it until 8.7.11_GA (assuming that's next), no?  If so, we'd want to change that.

Another option would be to stop guessing, and just run mysql_upgrade every time, as I believe it is supposed to be OK to run regardless of whether or not it is necessary.  With that said, upgrades tend to be slow still. Perhaps it's not ideal to run in large environments even when unnecessary?  And a guess like this is better than a release note to tell admins they should run mysql_upgrade during a maintenance window?